src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss

```diff
@@ -13,14 +13,20 @@
 }
 
 .withBackBackButton {
-  justify-content: space-between;
```

Comment:
The same as above. Desktop first, and then add `@include Mobile` with some styles for mobile :)